### ISSUE TITLE
sed: RE error: illegal byte sequence

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#546]
Hi,

If it's ok, that "RE error" needs to be removed to avoid confusion.
```
diffing r4.dat.save with r4.dat
PASSED
==============================================================
  Gzipped output data file comparison requires Linux OS.
  Skipped test: Gzipped output data file comparison
sed: RE error: illegal byte sequence
sed: RE error: illegal byte sequence
diffing a3.dat.bz2.save with a3.dat.bz2
PASSED
==============================================================
diffing r2.dat.save with r2.dat
PASSED
```

**Comment by drroe** [comment]
Yet another reason for me to bypass dacdif...

**Comment by drroe** [comment]
Addressed by #547.
